extras/rya.merger/src/test/java/org/apache/rya/accumulo/mr/merge/RulesetCopyIT.java

```diff
@@ -144,32 +146,33 @@ private static RyaType literal(final String lit, final URI type) {
 
     @BeforeClass
     public static void setUpPerClass() throws Exception {
+        DemoUtilities.setupLogging(LoggingDetail.LIGHT);
         accumuloDualInstanceDriver = new AccumuloDualInstanceDriver(IS_MOCK, true, true, false, false);
         accumuloDualInstanceDriver.setUpInstances();
     }
 
     @Before
     public void setUpPerTest() throws Exception {
-        parentConfig = accumuloDualInstanceDriver.getParentConfig();
-        childConfig = accumuloDualInstanceDriver.getChildConfig();
-        accumuloDualInstanceDriver.setUpTables();
         accumuloDualInstanceDriver.setUpConfigs();
+        accumuloDualInstanceDriver.setUpTables();
         accumuloDualInstanceDriver.setUpDaos();
+        parentConfig = accumuloDualInstanceDriver.getParentConfig();
+        childConfig = accumuloDualInstanceDriver.getChildConfig();
         parentDao = accumuloDualInstanceDriver.getParentDao();
     }
 
     @After
     public void tearDownPerTest() throws Exception {
         log.info("tearDownPerTest(): tearing down now.");
-        accumuloDualInstanceDriver.tearDownDaos();
         accumuloDualInstanceDriver.tearDownTables();
+        accumuloDualInstanceDriver.tearDownDaos();
+        if (rulesetTool != null) {
+            rulesetTool.shutdown();
+        }
     }
 
     @AfterClass
     public static void tearDownPerClass() throws Exception {
-        if (rulesetTool != null) {
-            rulesetTool.shutdown();
-        }
         accumuloDualInstanceDriver.tearDown();
```

Comment:
I can verify that it this helps. Ran tests from the command line ("mvn verify"), monitored the number of org.apache.accumulo.start.Main processes running: seems to vary between 11 (during tests) down to 5 (between tests), where previously it continued to climb up to a max of 31 for me. And the change makes sense: the tool gets initialized in each test (initialization happens in runRulesetCopyTest, which each test calls), so it should be torn down after each test.